dataactcore/utils/timeStampMixin.py

```diff
@@ -1,17 +1,17 @@
-from datetime import datetime
+from datetime import datetime, UTC
+
 from sqlalchemy import Column, DateTime, event
 
 
 class TimeStampMixin(object):
-    """ Timestamping mixin
-    """
-    created_at = Column(DateTime, default=datetime.utcnow)
-    updated_at = Column(DateTime, default=datetime.utcnow)
+    """ Timestamping mixin """
+    created_at = Column(DateTime, default=lambda: datetime.now(UTC).replace(tzinfo=None))
+    updated_at = Column(DateTime, default=lambda: datetime.now(UTC).replace(tzinfo=None))
 
     @staticmethod
     def _updated_at(mapper, connection, target):
         if not getattr(target, "ignore_updated_at", None):
-            target.updated_at = datetime.utcnow()
+            target.updated_at = datetime.now(UTC).replace(tzinfo=None)
```

Comment:
Does the `TimeStampMixin` not work if you use `get_utc_now()`?

Reply:
It causes circular imports because this base is imported into every single model so we can't then import models into function bag if we do that